builds/checkin/dotnet.yaml

```diff
@@ -63,11 +63,7 @@ jobs:
     pool:
       vmImage: "vs2017-win2016"
     steps:
-      - task: PowerShell@2
-        displayName: Install
-        inputs:
-          filePath: scripts/windows/setup/Install-Prerequisites.ps1
-          arguments: -DotnetSdkUrl $(NetCorePackageUriWindows) -Dotnet -Nuget
+      - template: ../templates/install-dotnet3.yaml
```

Comment:
Just curious, since ms hosted agent (vs2017-win2016) already have dotnet installed.  Does it detect and do nothing? or it will still install and over-write?

Reply:
It detects it and uses the found version.